src/hooks/useAnalyticsTokenInput.ts

```
@@ -0,0 +1,262 @@
+      await serverAnalytics.track('input_focused', {
+        input_type: inputType,
+        token_symbol: tokenSymbol,
+        timestamp: Date.now(),
+      })
+    } catch (error) {
+      console.error('Analytics: Failed to track input focus:', error)
+    }
+  }, [enabled, trackInputFocus])
+
+  // Track input blur
+  const trackInputBlur = useCallback(async (inputType: 'amount' | 'token', tokenSymbol?: string, finalAmount?: string) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'trackInputBlur'. Consider to delete it or rename it.

'trackInputBlur' is defined here:


(lint/suspicious/noRedeclare)